lib/app.js

```diff
@@ -25,19 +28,21 @@ exports.getDocuments = getDocuments = function(params, callback) {
   });
 }
 
-var options = {
-  hostname: 'localhost',
-  port: 8984,
-  path: '/CSD/documents.json',
-  agent: false
+exports.createChannelConfigObject = createChannelConfigObject = function(name, callback) {
+  var channelConfig = {};
+  fs.readFile('resources/channelConfig.json', function read(err, data) {
```

Comment:
Here you can just do `require('resources/channelConfig.json')` to get a json object.